docs/book/src/plugins/multi-module-v1-alpha.md

```diff
@@ -0,0 +1,78 @@
+# Multi Module Plugin (multi-module/v1-alpha)
+
+The multi-module plugin allows users to modify the existing scaffolding setup of kubebuilder to enable [Monorepo][monorepo] support. It abstracts the complexities to achieve this goal by easing the most difficult part of getting into Monorepos, namely the setup and the scaffolding required.
+
+By using this plugin you will have:
+
+- Separate go.mod modules in your API definition and your controller
+- an adjusted and optimized Image Building in your `Dockerfile` for dealing with Monorepos
+- an updated main module that has correct `replace` [directives][replace-directives] in place for developing locally.
+- Ability to switch from Monorepo to Classic Repository structure at will.
+
+<aside class="note">
+<h1>Examples</h1>
+
+See the "project-v3-multimodule" directory under the [testdata][testdata] directory of the Kubebuilder project to check an example of a scaffolding created using this plugin.
+
+</aside>
+
+## When to use it ?
+
+- If you want to reduce impact of transitive dependencies on your API being included in other projects
+- If you are looking to separately manage the lifecycle of your API release process from your controller release process.
+- If you are looking to modularize your codebase without splitting your code between multiple repositories.
+
+## How to use it ?
+
+After you create a new project with `kubebuilder init --plugins multi-module` you can create modularized APIs using this plugin. Ensure that you have followed the [quick start](https://book.kubebuilder.io/quick-start.html) before trying to use it.

```

Comment:
I did not understood your question @jmrodri.
That is a plugin that can be used with go/v3 or go/v4-alpha howevever

And to use you can:

kubebuilder init --plugins go/v3,multi-module.
AND
kubebuilder init --plugins go/v4-alpha/multi-module <enable>=true (@jakobmoellersap is that right? we need to proper clarify the option as well)

OR
kubebuilder init 
kubebuilder edit --plugins multi-module --enable=true

OR 

kubebuilder init  --plugins go/v4
kubebuilder edit --plugins multi-module --enable=true

Reply:
I think that first sentence may be a tad misleading for some. 
It says:
> After you create a new project with `kubebuilder init --plugins multi-module`

This may convey the idea that the ONLY plugin you need to use is `multi-module` when using `kubebuilder init` but that is not the case. We may want to update this part to include the explicit usage information that @camilamacedo86 mentioned.

Reply:
Also, one question should I use or not the flag with the init as well right?